mappings/net/minecraft/world/dimension/AreaHelper.mapping

```diff
@@ -9,12 +9,29 @@ CLASS net/minecraft/class_2424 net/minecraft/world/dimension/AreaHelper
 	FIELD field_25883 IS_VALID_FRAME_BLOCK Lnet/minecraft/class_4970$class_4973;
 	METHOD <init> (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350$class_2351;)V
 		ARG 1 world
+		ARG 2 pos
 		ARG 3 axis
 	METHOD method_10359 validStateInsidePortal (Lnet/minecraft/class_2680;)Z
+		ARG 0 state
 	METHOD method_10360 isValid ()Z
 	METHOD method_10362 wasAlreadyValid ()Z
 	METHOD method_10363 createPortal ()V
+	METHOD method_30485 getNewPortal (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350$class_2351;)Ljava/util/Optional;
+		ARG 0 world
+		ARG 1 pos
+		ARG 2 axis
+	METHOD method_30486 getOrEmpty (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;Ljava/util/function/Predicate;Lnet/minecraft/class_2350$class_2351;)Ljava/util/Optional;
+		ARG 0 world
+		ARG 1 pos
+		ARG 3 axis
 	METHOD method_30487 (Lnet/minecraft/class_2680;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Z
 		ARG 0 state
 		ARG 1 world
 		ARG 2 pos
+	METHOD method_30492 getLowerCorner (Lnet/minecraft/class_2338;)Lnet/minecraft/class_2338;
```

Comment:
Honestly this class needs to be revamped

Reply:
Out of scope for this PR